client.go

```diff
@@ -286,10 +297,10 @@ func (c *Client) Transaction(ctx context.Context, ID string) (*response.Transact
 	return res, nil
 }
 
-// XPub retrieves the complete xpub information for the current user.
+// XPubAPI retrieves the complete xpub information for the current user.
 // The server's response is expected to be unmarshaled into a *response.Xpub struct.
 // If the request fails or the response cannot be decoded, an error is returned.
-func (c *Client) XPub(ctx context.Context) (*response.Xpub, error) {
+func (c *Client) XPubAPI(ctx context.Context) (*response.Xpub, error) {
```

Comment:
it was intentional as it refer to the name of field in Client struct .. and allowing me to add method to return xPub ..
Why this was called before XPub if is referencing some XPubAPI ? what is XPubAPI
`res, err := c.xpubAPI.XPub(ctx)`

Reply:
It was named XPub because it reflects the response type returned after calling the API. In my view, the change you proposed could lead to ambiguity, as it implies that the method would return a specific API implementation, like a client with methods dedicated to handling xPub API calls.

@chris-4chain  @dorzepowski Please take a look at this as well, because it’s possible that the original naming wasn’t quite appropriate, and we might want to consider making some changes :)

Reply:
however i reverted this name and used new names in my methods with prefix `Get`

Reply:
You haven't apply the changes even though you liked by comment above and resolved the conversation. 🤷

Reply:
sorry i thought it was done ..